hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -75,7 +79,12 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
       buckets: 0,
       keys: 0,
       missingContainersCount: 0,
-      lastUpdated: 0
+      lastUpdated: 0,
```

Comment:
Maybe change here as well to refer that this is the refresh timstamp.
```suggestion
      lastRefreshed: 0,
```

Reply:
Done Changes as Suggested by You.